### ISSUE TITLE
ComponentGroup contains Components

### ISSUE THREAD

**Issue by BoogalooJB** [InseeFr/Pogues-Model#6]
Do we really want a `ComponentGroup` to be as sequence of `ComponentType` elements ? That would mean that we need to serialize every component in multiple places : in the sequence it belongs to, and in the component groups where it is referenced, wouldn't it ?

Might we use an `xs:token` element like we do for `codeListReference` or `ifTrue` element, juste to keep track of the `id` of the component ?

```xml
<xs:element name="ComponentGroup">
  <xs:complexType>
    <xs:sequence>
      <xs:element name="Name" type="xs:token"/>
      <xs:element name="Label" type="xs:token" maxOccurs="unbounded"/>
      <xs:element name="Declaration" type="DeclarationType" minOccurs="0" maxOccurs="unbounded"/>
      <xs:element name="Member" type="ComponentType" maxOccurs="unbounded"/>
    </xs:sequence>
    <xs:attribute name="id" type="xs:ID" use="required"/>
  </xs:complexType>
</xs:element>
````

**Comment by ThomasPO** [comment]
I think too that it's better to track the `id`s of the components

**Comment by FranckCo** [comment]
In some cases it make sense to have full components (e.g. components imported from repository and not yet placed on the questionnaire), so I suggest to have both a list of members (which could be empty) and a list of member references.

**Comment by BoogalooJB** [comment]
Ok.

**Comment by FranckCo** [comment]
Closed with commit 54c10b794e0109424bbf94af14ceed6d6d1a7d6e
